Snowflake.Data.Tests/UnitTests/SFS3ClientTest.cs

```diff
@@ -219,6 +220,36 @@ public void TestUploadFile(string requestKey, ResultStatus expectedResultStatus)
             AssertForUploadFileTests(expectedResultStatus);
         }
 
+        [Test]
+        public void SetCommonClientConfig_AppendHttpsToURL()
```

Comment:
In this project we follow the convention that the test name starts with "Test" and we don't use underscore in test names.

Reply:
fixed in [9bb006b](https://github.com/snowflakedb/snowflake-connector-net/pull/914/commits/9bb006b2ba4c6437e4a7165b34ba9aed6cdcc5fc)

Reply:
now it's Ok, but I would even remove SetCommonClientConfig part because is not necessary.

Reply:
I removed `SetCommonClientConfig` in  [e0f2247](https://github.com/snowflakedb/snowflake-connector-net/pull/914/commits/e0f2247f1afa30ef7a057255988b19127580604e)